app/models/flags/actions/action.rb

```diff
@@ -0,0 +1,31 @@
+class Flags::Actions::Action
```

Comment:
Thanks Josh, I see what you mean about it being confusing. `Base` is a really good suggestion, I like that much better as well! 

Plus, I've been thinking I might need a `Flags::Action` value object to avoid all the hash fetching in the views. But thought that might have been one class named "action" too many lol. This will free that up 🚀